.github/workflows/Build.yml

```diff
@@ -46,6 +46,14 @@ jobs:
           cache-dependency-path: "**/package-lock.json"
       - name: "Installing dependencies"
         run: npm run bootstrap
+      - name: Cache Nx
+        uses: actions/cache@c3f1317a9e7b1ef106c153ac8c0f00fed3ddbc0d
+        with:
+          path: node_modules/.cache/nx
+          key: cache-nx-${{ hashFiles('yarn.lock') }}-${{ matrix.target }}-${{ matrix.jobIndex }}-${{ github.sha }}
+          restore-keys: |
+            cache-nx-${{ hashFiles('yarn.lock') }}-${{ matrix.target }}-${{ matrix.jobIndex }}-
+            cache-nx-${{ hashFiles('yarn.lock') }}-
```

Comment:
Why this two ends with `-`?

Reply:
And where this `yarn.lock` comes from?

Reply:
You're right, this was a poor copy-paste. I've adjusted it now.